open_lm/params.py

```diff
@@ -90,6 +90,8 @@ def parse_args(args):
     )
     parser.add_argument(
         "--dataset-metadata",
+        type=str,
+        nargs="+",
```

Comment:
are multiple manifest files supported (as suggested by the `nargs="+"` flag)? looking at `get_metadata_file(...)` it looks like only 1 file can be passed

Reply:
Multiple manifest files are supported - ```get_metadata_file``` is just called separately on each.